packages/core/admin/admin/src/pages/MarketplacePage/PluginCard/index.js

```diff
@@ -1,185 +1,110 @@
+const PluginLogo = styled.img`
+  display: block;
+  width: 64px;
+  height: 64px;
+  object-fit: contain;
+  border-radius: 6px;
+`;
+
+const PluginDescription = styled.p`
+  /* stylelint-disable value-no-vendor-prefix, property-no-vendor-prefix */
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 2;
+  /* stylelint-enable value-no-vendor-prefix, property-no-vendor-prefix */
+  overflow: hidden;
+`;
+
+const PluginCard = ({ plugin }) => {
+  const { id, attributes } = plugin;
+  const { formatMessage } = useIntl();
+
+  return (
+    <Box
+      paddingLeft={6}
+      paddingRight={6}
+      paddingTop={4}
+      paddingBottom={4}
+      hasRadius
+      background="neutral0"
+      key={id}
+      shadow="tableShadow"
+      style={{ height: '100%' }}
+    >
+      <Flex
+        justifyContent="space-between"
+        direction="column"
+        alignItems="flex-end"
+        style={{ height: '100%' }}
+      >
+        <div style={{ width: '100%' }}>
+          <PluginLogo className="logo" src={attributes.logo.url} alt={`${attributes.name} logo`} />
+          <Box paddingTop={5}>
+            <Typography variant="delta" className="name">
+              {attributes.name}
+            </Typography>
+          </Box>
+          <Box paddingTop={3}>
+            <PluginDescription>
+              <Typography variant="omega" className="description" textColor="neutral600">
+                {attributes.description}
+              </Typography>
+            </PluginDescription>
+          </Box>
         </div>
-
-        <PopUpWarning
-          content={{
-            message: 'app.components.PluginCard.PopUpWarning.install.impossible.autoReload.needed',
-            title: 'app.components.PluginCard.PopUpWarning.install.impossible.title',
-            confirm: 'app.components.PluginCard.PopUpWarning.install.impossible.confirm',
-          }}
-          isOpen={this.state.showModalAutoReload}
-          onlyConfirmButton
-          onConfirm={() => this.setState({ showModalAutoReload: false })}
-          popUpWarningType="warning"
-        />
-        <PopUpWarning
-          content={{
-            message: 'app.components.PluginCard.PopUpWarning.install.impossible.environment',
-            title: 'app.components.PluginCard.PopUpWarning.install.impossible.title',
-            confirm: 'app.components.PluginCard.PopUpWarning.install.impossible.confirm',
-          }}
-          isOpen={this.state.showModalEnv}
-          onlyConfirmButton
-          onConfirm={() => this.setState({ showModalEnv: false })}
-          popUpWarningType="warning"
-        />
-      </Wrapper>
-    );
-  }
-}
-
-PluginCard.defaultProps = {
-  isAlreadyInstalled: false,
-  plugin: {
-    description: '',
-    id: '',
-    name: '',
-    price: 0,
-    ratings: 5,
-  },
+        <Stack horizontal size={2} paddingTop={3}>
+          <LinkButton
+            size="S"
+            href={`https://market.strapi.io/plugins/${attributes.slug}`}
```

Comment:
I think you might need to add the `target="_blank"` prop here combined with the `rel="noopener noreferrer"` one.

Reply:
I this is already done by the component: https://github.com/strapi/design-system/blob/main/packages/strapi-design-system/src/LinkButton/LinkButton.js#L47-L48